project/target/maven-archiver/pom.properties

```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
```

Comment:
It seems that you might be missing a `.gitignore' file somewhere, as I would expect this generated code under target not to be included.

Reply:
Done